Code/Editor/Include/Command.h

```diff
@@ -8,23 +8,33 @@
 
 
 // Description : Classes to deal with commands
-
-
-#ifndef CRYINCLUDE_EDITOR_INCLUDE_COMMAND_H
-#define CRYINCLUDE_EDITOR_INCLUDE_COMMAND_H
 #pragma once
 
 #include <QString>
-
+#include <AzCore/std/containers/vector.h>
+#include <AzCore/std/string/conversions.h>
+#include <CryCommon/LegacyAllocator.h>
 #include "Util/EditorUtils.h"
 
 inline AZStd::string ToString(const QString& s)
 {
     return s.toUtf8().data();
 }
 
+
 class CCommand
 {
+    static inline bool FromString(int32 &val, const char* s) {
+        if(!s)
+        {
+            return false;
+        }
+        val = (int)strtol(s, nullptr, 10);
```

Comment:
preffer to use aznumeric_cast instead of casting to int, this notifies of possible conversion loss in runtime.